src/test/java/com/simpligility/maven/plugins/android/sample/AbstractSampleIT.java

```diff
@@ -0,0 +1,58 @@
+/*
```

Comment:
Could you remove that class its usage. It hides too much detail and makes the tests harder to read imho.